### DISCUSSION TITLE
City boundaries

### DISCUSSION THREAD

**Discussion by AnnePdM** [koenderks/rcityviews#21] [Q&A]
Is there anyway to expand the city boundaries outward? For example, if I wanted to map Amsterdam and Amstelveen together?

**Comment by koenderks** [comment]
Did you try lowering the value of the `zoom` argument below 1? Let me know if that does the trick for you. Note that this will increase the computation time.

**Reply by AnnePdM** [reply]
Thanks. That did work. I made the mistake of only looking at the preview in RStudio. When I saved the picture, the map expanded further.
